package.json

```diff
@@ -13,7 +13,7 @@
     "build-packages": "preconstruct build",
     "build": "./scripts/build.sh",
     "vercel-build": "echo \"you're welcome - nothing to build\"",
-    "generate-icons": "yarn workspaces foreach -p --include @commercetools-docs/gatsby-theme-docs,@commercetools-docs/ui-kit,@commercetools-website/docs-smoke-test run generate-icons && prettier --write '**/generated/*.js'",
+    "generate-icons": "yarn workspaces foreach -p --include @commercetools-docs/gatsby-theme-docs,@commercetools-website/docs-smoke-test,@commercetools-docs/ui-kit run generate-icons && prettier --write '**/generated/*.{js,ts,tsx}'",
     "lint": "jest --projects jest.eslint.config.js jest.stylelint.config.js jest.text.config.js",
     "lint:js": "jest --config jest.eslint.config.js",
     "lint:css": "jest --config jest.stylelint.config.js",
```

Comment:
We should also add a `typecheck` command: https://github.com/commercetools/ui-kit/blob/c79b62bd25cd3da1ffe7fd9e39d08b8335218942/package.json#L41

And include it in CI: https://github.com/commercetools/ui-kit/blob/c79b62bd25cd3da1ffe7fd9e39d08b8335218942/.github/workflows/main.yml#L43-L44

And in lint-staged: https://github.com/commercetools/ui-kit/blob/c79b62bd25cd3da1ffe7fd9e39d08b8335218942/lint-staged.config.js#L17-L29

Reply:
+1